signac/contrib/filterparse.py

```diff
@@ -53,7 +131,17 @@ def _parse_json(q):
 
 
 def _cast(x):
-    "Attempt to interpret x with the correct type."
+    """Attempt to interpret x with the correct type.
+
+    Parameters
+    ----------
+    x :
+
+
+    Returns
+    -------
+
```

Comment:
I'm not sure what to put here. Can you find an example of a NumPy docstring that doesn't have a specified return type?

Reply:
Here is the example for different type return in a methods that I following. [link](https://github.com/scipy/scipy/blob/master/scipy/cluster/vq.py#L363)

Reply:
Unfortunately that example is for a slightly different case. That would apply if `_cast` returned _two_ values. This function doesn't have a defined return type. I will make a suggestion above.